examples/resources/awscc_backup_backup_selection/second_example.tf

```
@@ -0,0 +1,12 @@
+resource "awscc_backup_backup_selection" "example2" {
```

```suggestion
resource "awscc_backup_backup_selection" "example" {
```
there's no need to put increment in the resource name, we assume each examples are atomic